File: src/Book.java
public class Book {
    private String title;
    private User renter;

    public Book(String title) {
        this.title = title;
    }

    public String getTitle() {
        return title;
    }

    public User getRenter() {
        return renter;
    }

    public void setRenter(User renter) {
        this.renter = renter;
    }
}

File: src/src/book.py
class Book:
    def __init__(self, title):
        self.title = title
        self.renter = None

    def get_title(self):
        return self.title

    def get_renter(self):
        return self.renter

    def set_renter(self, renter):
        self.renter = renter
